write-mcp-output.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+OUTPUT_FILE="mcp-output.txt"
+
+echo "Current UTC time is 2025-07-28 01:23:46, and the time in America/New_York is 2025-07-27 21:23:46." > "$OUTPUT_FILE" 2>/dev/null
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)